# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hyprland/egl.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- hyprland-source/CMakeLists.txt.vanilla	2023-12-22 12:21:52.141500820 +0100
+++ hyprland-source/CMakeLists.txt	2023-12-22 12:22:40.513497233 +0100
@@ -222,7 +222,8 @@
 
 target_link_libraries(Hyprland
         ${CMAKE_SOURCE_DIR}/subprojects/wlroots/build/libwlroots.so.13032 # wlroots is provided by us
-        OpenGL::EGL
+        EGL
+        GLESv2
         OpenGL::GL
         Threads::Threads
         libudis86
